lib/manager/components/DeploymentSettings.js

```diff
@@ -157,23 +180,44 @@ export default class DeploymentSettings extends Component {
     }
   }
 
+  _noEdits = () => Object.keys(this.state)
+    .map(key => shallowEqual(this.state[key], this.props.project[key]))
+    .indexOf(false) === -1
```

Comment:
Use [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) instead of map + indexOf